requirements.txt

```diff
@@ -1,3 +1,5 @@
 numpy
 networkx
-tqdm
\ No newline at end of file
+tqdm
+lenskit==0.11.1
+pandas==1.0.5
```

Comment:
new dependencies on `lenskit` and `pandas` (the latter is required for the `lenskit` MF fitting algo)